app/src/main/java/com/github/sdpteam15/polyevents/database/DatabaseConstant.kt

```diff
@@ -11,6 +11,7 @@ object DatabaseConstant {
         EVENT_COLLECTION("events"),
         PROFILE_COLLECTION("profile"),
         USER_COLLECTION("users"),
+        EVENT_ATTENDEES_COLLECTION("eventAttendees");
```

Comment:
Not sure to see why you choose to create another table instead of having an array of participant in the event document

Reply:
Yes sorry! This is not used anymore. I will remove it. Good find.